src/analysis/retail/evoker/preservation/modules/talents/EssenceBurst.tsx

```diff
@@ -11,17 +11,27 @@ import { SPELL_COLORS } from 'analysis/retail/evoker/preservation/constants';
 import DonutChart from 'parser/ui/DonutChart';
 import { SpellLink } from 'interface';
 import { t } from '@lingui/macro';
+import ItemManaGained from 'parser/ui/ItemManaGained';
 
 const ESSENCE_COSTS: { [name: string]: number } = {
   'Emerald Blossom': 3,
   Echo: 2,
   Disintegrate: 3,
 };
 
+const MANA_COSTS: { [name: string]: number } = {
```

Comment:
The mana costs of these abilities should be a property on their SPELLS declaration